whois-commons/src/main/resources/nrtm_schema.sql

```diff
@@ -19,58 +35,72 @@ create table `source`
 ) ENGINE = InnoDB
   DEFAULT CHARSET = utf8;
 
-DROP TABLE IF EXISTS `version_information`;
-create table `version_information`
+CREATE TABLE `version_info`
 (
     `id`             int unsigned NOT NULL AUTO_INCREMENT,
     `source_id`      int unsigned NOT NULL,
-    `version`        int unsigned NOT NULL DEFAULT '0',
-    `session_id`     varchar(128) NOT NULL DEFAULT '',
-    `type`           varchar(128) NOT NULL DEFAULT '',
-    `last_serial_id` int          NOT NULL DEFAULT '0',
+    `version`        int unsigned NOT NULL,
+    `session_id`     varchar(128) NOT NULL,
+    `type`           varchar(128) NOT NULL,
+    `last_serial_id` int          NOT NULL,
     PRIMARY KEY (`id`),
-    CONSTRAINT `version_information__source_fk` FOREIGN KEY (`source_id`) REFERENCES `source` (`id`),
-    UNIQUE KEY `version_information__source__version_uk` (`source_id`, `version`),
-    UNIQUE KEY `version_information__session__version_uk` (`session_id`, `version`),
-    UNIQUE KEY `version_information__type__last_serial_id_uk` (`type`, `last_serial_id`)
+    CONSTRAINT `version__source__fk` FOREIGN KEY (`source_id`) REFERENCES `source` (`id`),
+    UNIQUE KEY `version__source__version__uk` (`source_id`, `version`),
+    UNIQUE KEY `version__session__version__uk` (`session_id`, `version`),
+    UNIQUE KEY `version__type__last_serial_id__uk` (`type`, `last_serial_id`)
 ) ENGINE = InnoDB
   DEFAULT CHARSET = utf8;
 
-DROP TABLE IF EXISTS `snapshot_file`;
-create table `snapshot_file`
+CREATE TABLE `snapshot_object`
```

Comment:
Consider renaming from `snapshot_object` since the object data can be included in delta file also?

Reply:
OK